backend/src/main/java/com/bakdata/conquery/sql/conversion/cqelement/concept/CQConceptConverter.java

```diff
@@ -278,26 +279,34 @@ private CQTableContext createTableContext(TablePath tablePath, CQConcept cqConce
 
 		SqlIdColumns ids = convertIds(cqConcept, cqTable, conversionContext);
 		ConnectorSqlTables connectorTables = tablePath.getConnectorTables(cqTable);
-		Optional<ColumnDateRange> tablesValidityDate = convertValidityDate(cqTable, connectorTables.getLabel(), conversionContext);
+		ValidityDate validityDate = cqTable.findValidityDate();
+
+		Optional<ColumnDateRange> tablesValidityDate = convertValidityDate(validityDate, connectorTables.getLabel(), conversionContext);
```

Comment:
Ich mag die Kommentare für die Struktur im Rest der Funktion 👍 
```suggestion
		ConnectorSqlTables connectorTables = tablePath.getConnectorTables(cqTable);

		// convert concept ids
		SqlIdColumns ids = convertIds(cqConcept, cqTable, conversionContext);

		// convert validity date
		ValidityDate validityDate = cqTable.findValidityDate();
		Optional<ColumnDateRange> tablesValidityDate = convertValidityDate(validityDate, connectorTables.getLabel(), conversionContext);
```